### PR TITLE
Various small fixes

### PR DESCRIPTION
**Pull request by TAdev0**
This change is [Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/670)

### CODE REVIEW

File: src/ch01-01-installation.md [hunk 1]
```diff
@@ -47,8 +47,8 @@ Otherwise, you can simply run the following command in your terminal, and follow
 ```bash
 curl --proto '=https' --tlsv1.2 -sSf https://docs.swmansion.com/scarb/install.sh | sh
 ```
-
-- In both cases, you can verify installation by running the following command in a new terminal session, it should print both Scarb and Cairo language versions, e.g:
+<br>
```
Comment (enitrat) [inline src/ch01-01-installation.md:50]: line break? if yes just skip one extra line in the md

File: src/ch02-02-data-types.md [hunk 2]
```diff
@@ -70,15 +69,17 @@ that number literals that can be multiple numeric types allow a type suffix,
...
-<span class="caption">Table 3-2: Integer Literals in Cairo</span>
-
 | Numeric literals | Example   |
 | ---------------- | --------- |
 | Decimal          | `98222`   |
 | Hex              | `0xff`    |
 | Octal            | `0o04321` |
 | Binary           | `0b01`    |
 
+<br>
```
Comment (enitrat) [inline src/ch02-02-data-types.md:79]: skip extra line instead

File: src/ch02-02-data-types.md [hunk 3]
```diff
@@ -53,6 +49,9 @@ the type of an integer value.
 | 256-bit | `u256`   |
 | 32-bit  | `usize`  |
 
+<br>
```
Comment (enitrat) [inline src/ch02-02-data-types.md:52]: same

File: src/ch12-02-anatomy-of-a-simple-contract.md [hunk 4]
```diff
@@ -15,10 +15,10 @@ Let's consider the following contract to present the basics of a Starknet contra
 
 ## What is this contract?
 
-Contracts are a combintation of state and logic.
-The state is defined within the `Storage` struct, and is always initialized empty. Here, our struct contains a single a field called `stored_data` of type `u128` (unsigned integer of 128 bits), indicating that our contract can store any number between 0 and \\( {2^{128}} - 1 \\).
+Contracts are a combination of state and logic.
+The state is defined within the `Storage` struct, and is always initialized empty. Here, our struct contains a single field called `stored_data` of type `u128` (unsigned integer of 128 bits), indicating that our contract can store any number between 0 and \\( {2^{128}} - 1 \\).
 The logic is defined by functions that interact with the state. Here, our contract defines and publicly exposes the functions `set` and `get` that can be used to modify or retrieve the value of the stored variable.
-You can think of it as a single slot in a database that you can query and modify by calling functions of the code that manages the database.
+You can think of it as a single slot in a database that you can query and modify by calling functions of the code that manage the database.
```
Comment (enitrat) [inline src/ch12-02-anatomy-of-a-simple-contract.md:21]: manages

**Review by enitrat — CHANGES_REQUESTED** [review]
> Reviewed 42 of 43 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/670)* status: all files reviewed, 1 unresolved discussion (waiting on @TAdev0)

___
*[`listings/ch09-error-handling/no_listing_01_panic/src/lib.cairo` line 3 at r2](https://reviewable.io/reviews/cairo-book/cairo-book/670#-NsZw3_pC4j4VnInfk-a:-NsZw3_pC4j4VnInfk-b:bxbp84d) ([raw file](https://github.com/cairo-book/cairo-book/blob/da756fc461ace0a0c409b32562153369f3d5d70e/listings/ch09-error-handling/no_listing_01_panic/src/lib.cairo#L3)):*
> ```Cairo
> fn main() {
>     let mut data = array![];
>     data.append(2);
> ```

can be merged in one line



_Code quote:_
```Cairo
    let mut data = array![];
    data.append(2);
```

**Review by TAdev0 — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/670)* status: all files reviewed, 1 unresolved discussion (waiting on @enitrat)

___
*[`listings/ch09-error-handling/no_listing_01_panic/src/lib.cairo` line 3 at r2](https://reviewable.io/reviews/cairo-book/cairo-book/670#-NsZw3_pC4j4VnInfk-a:-Ns_LEBh1mXs96xIiVlx:b-7rl0jq) ([raw file](https://github.com/cairo-book/cairo-book/blob/da756fc461ace0a0c409b32562153369f3d5d70e/listings/ch09-error-handling/no_listing_01_panic/src/lib.cairo#L3)):*
<details><summary><i>Previously, enitrat (Mathieu) wrote…</i></summary><blockquote>

can be merged in one line
</blockquote></details>

done!

**Review by enitrat — APPROVED** [review]
> Reviewed 5 of 5 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cairo-book/cairo-book/670)* status: <span>:shipit:</span> complete! all files reviewed, all discussions resolved (waiting on @TAdev0)

**Pull request merged**